src/templates/base/2019/index.html

```diff
@@ -24,70 +24,74 @@
   <link rel="stylesheet" href="/static/css/index.css?v=20200728001611">
 {% endblock %}
 
-{% block main %}
-<main id="maincontent">
-  <section id="intro" class="intro-container">
-    <div class="intro">
-      <div class="intro-year">{{ year }}</div>
-      <h1 class="title title-lg title-alt">{{ self.intro_title() }}</h1>
-      <h2>{{ self.intro_sub_title() }}</h2>
-      <p>{{ self.mission() }}</p>
-      <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}" class="btn">
-        {{ self.start_exploring() }}
-      </a>
-    </div>
-    <div class="intro-image-wrapper">
-      <svg viewBox="0 0 50 16">
-        <text x="7" y="15">{{ year }}</text>
-      </svg>      
-      <img src="/static/images/home-hero.png" alt="" width="562" height="820">
-    </div>
-  </section>
-  {% include "%s/2019/featured_chapters.html" % lang %}
-  <section id="contributors" class="contributors-container alt-bg">
-    <div class="contributors">
-      <h2 class="title title-alt">{{ self.contributors_title() }}</h2>
-      <p>{{ self.contributors_description() }}</p>
-      <a href="{{ url_for('contributors', year=year, lang=lang) }}" class="alt btn">
-        {{ self.contributors_link() }}
-      </a>
-    </div>
-    <div class="people" aria-hidden="true">
-      <div class="people-number">{{ self.contributors() }}</div>
-      <img id="character-markup" class="character" src="/static/images/character-markup.png" alt="" width="154" height="252" loading="lazy" />
-      <img id="character-star" class="character" src="/static/images/character-star.png" alt="" width="245" height="254" loading="lazy" />
-      <img id="character-hat" class="character" src="/static/images/character-hat.png" alt="" width="186" height="207" loading="lazy" />
-    </div>
-  </section>
-  <section id="methodology" class="methodology-container">
-    <div class="methodology">
-      <h2 class="title title-center">{{ self.methodology_title() }}</h2>
-      <div class="methodology-data">
-        <div class="websites-tested">
-          <div>
-            {{ self.methodology_stat_1_title() }}
+{% block container %}
+  {% block main %}
```

Comment:
Do we need the `{% block main %}` declaration? There’s nothing special about it AFAIK so I think overriding the `{% block container %}` should be enough. Should save you some indents and make this PR look a lot smaller.

Reply:
Done.